utils/api.js

```diff
@@ -0,0 +1,28 @@
+export const api = {
+    async call(endpoint, options) {
+        const response = await window.fetch(`api/${endpoint}`, { ...options });
+        const result = await response.json();
+
+        if (result.success === false) {
+            throw new Error(result.error);
+        }
+
+        return { ...result.data };
```

Comment:
Pareil ici, pas besoin du spread.

```suggestion
        return result.data;
```